### DISCUSSION TITLE
v3.0.0-rc.2

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#18386] [Announcements]
Release notes: https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.2

**Comment by ahku** [comment]
After upgrading my useState stuff stopped working, middleware didn't read the states etc. I spent half an hour before downgrading and everything worked as usual again. Just a heads up, don't have time to reproduce and narrow down the reason at the moment.
